### DISCUSSION TITLE
MODIS product MYD10A1 two tiles are missing:

### DISCUSSION THREAD

**Discussion by aftabhunzai** [microsoft/PlanetaryComputer#121] [General]
I am trying to access tiles v16h15 and v16h20 using following code: 
To access v16h15 tile I used bbox and location but both didn't work.
```
from pystac_client import Client
from datetime import datetime
catalog = Client.open('https://planetarycomputer.microsoft.com/api/stac/v1')
longitude=-79
latitude=-75
bbox = [-172.7631,-80, -58.4517,  -70]
location = shapely.geometry.Point(longitude,latitude)
collection="modis-10A1-061"
datetime='2007-12-23 07:00:10-06:00'
search = catalog.search(
            collections=[collection],
            bbox=bbox,
            #intersects=location,
            datetime=str(datetime),
        )
ic=search.get_all_items()
if(len(ic)>1):
    item_t = search.get_all_items()[0]
    print(item_t.id)
    item_a = search.get_all_items()[1]
    print(item_a.id)
```
It looks like both tiles are missing as I am getting different tiles. Please check and help.

**Comment by TomAugspurger** [comment]
I think https://planetarycomputer.microsoft.com/api/stac/v1/collections/modis-10A1-061/items/MYD10A1.A2007357.h15v16.061.2021081061440 is the tile you're looking for, right?

```python
from pystac_client import Client
from datetime import datetime
import shapely.geometry

catalog = Client.open('https://planetarycomputer.microsoft.com/api/stac/v1')
longitude=-79
latitude=-75
bbox = [-172.7631,-80, -58.4517,  -70]
location = shapely.geometry.Point(longitude,latitude)
collection="modis-10A1-061"
datetime='2007-12-23 07:00:10-06:00'

search = catalog.search(
            collections=[collection],
            datetime=str(datetime),
            query={
                "modis:vertical-tile": {"eq": 16},
                "modis:horizontal-tile": {"eq": 15},
            }
        )

items = search.item_collection()
len(items)
```

That returns two items, because the item seems to have been reprocessed at some point. I got 0 results with your `bbox` query, strangely. The geometry for this item does look a bit strange: https://planetarycomputer.microsoft.com/api/data/v1/item/map?collection=modis-10A1-061&item=MYD10A1.A2007357.h15v16.061.2021081061440

**Reply by aftabhunzai** [reply]
Yes, there is some issue I could not understand and it did not work for intersection. Thank you. Your solution worked for me.

**Reply by TomAugspurger** [reply]
Looks like some of the antimeridian code handling in stactools might have resulted in a bad Item geometry. We're looking into it and will post an update when it's fixed.
